pyproject.toml

```diff
@@ -310,11 +310,31 @@ curl = ">=8.4.0"
 
 [tool.coverage.run]
 # See note above on need to specify separate sources for pytest-coverage and coverage.
-source = ["src/pudl/", "test/integration/", "test/unit/"]
+# source = ["src/pudl/", "test/integration/", "test/unit/"]
+include = [
+  "src/pudl/**",
+  "test/integration/**",
+  "test/unit/**",
+  "*/site-packages/pudl/**",
+]
```

Comment:
What's going on with the change from source to include here?  Should these be identical to the `pudl_sources` below? Should the commented out `source` line be removed? Is the comment referring above still relevant? Does switching from source to include mean we don't need the `--cov=...` arguments above?